docs/docs/articles/github-actions.md

```diff
@@ -1,135 +1,91 @@
-
-Testkube uses your K8s context and access settings in order to interact with the cluster and tests resources, etc.
-
-In the next few sections, we will go through the process of Testkube and Helm (for Testkube's release deploy/upgrade) automations with the usage of GitHub Actions and GKE K8s.
-
 ## Testkube GitHub Action
 
-The testkube GitHub Action is available here <https://github.com/kubeshop/testkube-docker-action> and it enables running the Testkube CLI commands in a GitHub workflow.
-
-The following example shows how to create a test using the GitHub action; a more complex example can be found [here](https://github.com/kubeshop/helm-charts/blob/59054b87f83f890f4f62cf966ac63fd7e46de336/.github/workflows/testkube-docker-action.yaml).
-
-```yaml
-# Creating test
-- name: Create test
-  id: create_test
-  uses: kubeshop/testkube-docker-action@v1
-  with:
-    command: create
-    resource: test
-    namespace: testkube
-    parameters: "--type k6/script --name testkube-github-action"
-    stdin: "import http from 'k6/http';\nimport { sleep,check } from 'k6';\n\nexport default function () {\n  const baseURI = `${__ENV.TESTKUBE_HOMEPAGE_URI || 'https://testkube.kubeshop.io'}`\n  check(http.get(`${baseURI}/`), {\n    'check testkube homepage home page': (r) =>\n      r.body.includes('Your Friendly Cloud-Native Testing Framework for Kubernetes'),\n  });\n\n\n  sleep(1);\n}\n"
-```
-
-## Configuring Your GH Actions for Access to GKE
-
-To obtain set up access to a GKE (Google Kubernetes Engine) from GH (GitHub) actions, please visit the official documentation from GH: <https://docs.github.com/en/actions/deployment/deploying-to-google-kubernetes-engine>.
-
-1. Create a Service Account (SA).
-2. Save it into GH's **Secrets** of the repository.
-3. Run either `Helm` or `Kubectl kubtest` commands against the set up GKE cluster.
+The Testkube GitHub Action installs Testkube and enables running any [Testkube CLI](https://docs.testkube.io/cli/testkube) command in a GitHub workflow. It is available on Github Marketplace <https://github.com/marketplace/actions/testkube-action>.
+The action provides a flexible way to work with your pipeline and can be used with Testkube Cloud, Testkube Enterprise, and an open source Testkube platforms.
 
-## Main GH Action Section Configuration
-
-To install on Linux or MacOS, run:
+### Testkube Cloud CLI Setup
+The following example shows how to create and run a test using the GitHub action on the [Testkube Cloud](https://cloud.testkube.io/) instance. Please note that there are no additional steps needed to connect to the k8s cluster as all the necessary data are provided as inputs. We recommend to use [Github's secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) for this to enhance the security.
```

Comment:
We should tell people why how and where to create the `token` mentioned here :)

Reply:
by `token` you mean?